src/EditorFeatures/CSharpTest/Formatting/CodeCleanupTests.cs

```diff
@@ -196,8 +196,13 @@ protected static async Task AssertCodeCleanupResult(string expected, string code
                 var document = workspace.CurrentSolution.GetDocument(hostdoc.Id);
 
                 var codeCleanupService = document.GetLanguageService<ICodeCleanupService>();
+
+                var docOptions = await document.GetOptionsAsync(CancellationToken.None).ConfigureAwait(false);
+                var organizeUsingsSet = new OrganizeUsingsSet(docOptions);
+                var enabledDiagnostics = codeCleanupService.GetEnabledDiagnostics(docOptions);
```

Comment:
how about just having GetCodeCleanupOptions(OptionSet) which return CodeCleanUpOptions with both OrganizeUsingSet and enabledDiagnostics ?